microcloud/test/includes/microcloud.sh

```diff
@@ -475,7 +475,7 @@ reset_snaps() {
       snap enable microcloud > /dev/null 2>&1 || true
       snap start lxd > /dev/null 2>&1 || true
       snap start microcloud > /dev/null 2>&1 || true
-      snap refresh lxd --channel latest/stable --cohort=+
+      snap refresh lxd --cohort=+
```

Comment:
Can this be latest/edge to catch LXD regressions with MicroCloud usage ?
Similar to how we use latest edge of liblxc and dqlite in LXD's tests.